pkg/ephemeral/io/feeder.go

```diff
@@ -50,21 +55,48 @@ type AmphoraFeeder struct {
 // LoadFromSecretStoreAndFeed loads input parameters from Amphora.
 func (f *AmphoraFeeder) LoadFromSecretStoreAndFeed(act *Activation, feedPort string, ctx *CtxConfig) ([]byte, error) {
 	var data []string
+	inputs := []ActivationInput{}
 	client := f.conf.AmphoraClient
 	for i := range act.AmphoraParams {
-		osh, err := client.GetSecretShare(act.AmphoraParams[i])
+		osh, err := client.GetSecretShare(act.AmphoraParams[i], ctx.Spdz.ProgramIdentifier)
 		if err != nil {
 			return nil, err
 		}
+		policy := "_DEFAULT_POLICY_"
```

Comment:
read default policy name from config

Reply:
The config will contain the policy attached to the ephemeral container itself. If the default policy is to be used this is to be configured via this configuration parameter.
A policcy to use as a fallback option, in case the policy parameter is not provided in the config, one could use a policy defined for the whole VCP, but this is to be implemented in future if required. The fallback as of now - as used in the referenced line - will be hardcoded for the time beeing.
Anyways, replaced with default policy `carbynestack.def`